src/script/lua_api/l_vmanip.cpp

```diff
@@ -108,6 +108,11 @@ int LuaVoxelManip::l_set_data(lua_State *L)
 		lua_pop(L, 1);
 	}
 
+	// FIXME: in theory we should clear VOXELFLAG_NO_DATA here.
+	// However there is no way to tell which values Lua code has intended to set
+	// (if they were VOXELFLAG_NO_DATA before), and which were just not touched.
+	// This potentially causes problems with unloaded areas *shrug*
```

Comment:
Worth mentioning: `MMVManip::initialEmerge` is never called with `load_if_inexistent=false` (it's only used in a unittest and benchmarks). So this is only an issue outside of map. (=> If you write with a vmanip outside of map, you still read ignore/0.)